eqcorrscan/utils/plotting.py

```diff
@@ -1392,6 +1424,11 @@ def pretty_template_plot(template, background=False, picks=False, **kwargs):
         axis.set_ylabel('.'.join([tr.stats.station, tr.stats.channel]),
                         rotation=0, horizontalalignment='right')
         axis.yaxis.set_ticks([])
+        # If an event is given, but no picks are specifically supplied, then
```

Comment:
If you go with the suggestion at the top (to remove the `picks` argument) then this bit can be removed.

Reply:
Thanks for this, I fixed it as suggested.